test/test_quantized_models.py

```diff
@@ -0,0 +1,89 @@
+import torch
+import unittest
+import torchvision
+
+from common_utils import TestCase, map_nested_tensor_object
+from collections import OrderedDict
+from itertools import product
+import torch
+import numpy as np
+from torchvision import models
+import unittest
+import traceback
+import random
+
+
+def set_rng_seed(seed):
+    torch.manual_seed(seed)
+    random.seed(seed)
+    np.random.seed(seed)
+
+
+quantizable_models = ['googlenet', 'inception_v3', 'shufflenet_v2_x0_5',
```

Comment:
We will need to merge these changes when the PR is approved, but otherwise looks good.
It also looks like so far all quantizable models are scriptable. If thats the case, lets remove the scriptable quantizable models and reuse the quantizable_models.

Reply:
May still need scriptable quantizable models, seems shufflenet has following error when scripting if use prepare_qat():
```
> RuntimeError:
> lhs of assignment must be a variable, subscript, or starred expression:
> at /data/users/haixin/fbsource/fbcode/buck-out/dev/gen/pytorch/vision/test/torchvision_quantized_models#binary,link-tree/torch/quantization/fake_qua
ntize.py:81:12
>         if self.observer_enabled:
>             self.observer(X.detach())
>             self.scale, self.zero_point = self.calculate_qparams()
>             ~~~~~~~~~~ <--- HERE
>         if self.fake_quant_enabled:
>             if self.qscheme == torch.per_channel_symmetric or self.qscheme == torch.per_channel_affine:
> 'FakeQuantize.forward' is being compiled since it was called from 'QuantizableInvertedResidual.forward'
> at /data/users/haixin/fbsource/fbcode/buck-out/dev/gen/pytorch/vision/test/torchvision_quantized_models#binary,link-tree/torchvision/models/quantiza
tion/shufflenetv2.py:33:32
>             out = self.cat.cat((x1, self.branch2(x2)), dim=1)
>         else:
>             out = self.cat.cat((self.branch1(x), self.branch2(x)), dim=1)
>                                 ~~~~~~~~~~~~~~ <--- HERE
>
>         out = shufflenetv2.channel_shuffle(out, 2)
> 'QuantizableInvertedResidual.forward' is being compiled since it was called from 'QuantizableShuffleNetV2.forward'
```

Reply:
Can you help prepare a minimal repro of this issue? Does this happen with prepare() or only with prepare_qat? We only check for scripting after convert, so the behavior with prepare and prepare_qat is identical, as models get quantized prior to scripting. It looks like the model to script contains fake-quant modules which might be causing the error.  Please file an issue in github once you have a minimal example. For now, lets exclude shufflenet from the list of scriptable models.

Reply:
FYI `shufflenet` is supposed to be scriptable, and we do have scriptability tests that pass for it. cc @eellison if the module dedup PR might have had any effect on this

Reply:
I did some tests and found the original float quantizable model is scriptable if don't do prepare/prepare_qat and convert. So the issue is not related to the float shufflenet model.

The root cause for the issues with prepare/prepare_qat are when branch1 is an empty Sequential(), the observer in branch1 is not removed during convert, resulting the error during scripting since observers are not scriptable. Will file an issue for that.

Reply:
This is a known issue https://github.com/pytorch/pytorch/issues/28303

The workaround should be pretty simple: 
```
tmp1, tmp2 = self.calculate_qparams()`
self.scale = tmp1
self.zero_point = tmp2
```

Reply:
Update: The error is fixed in @raghuramank100 's PR: #28384, quantized shufflenetv2 can be scripted now.